mocha.opts

```diff
@@ -1,2 +1,2 @@
 --compilers js:babel/register
-@(components|widgets|lib|decorators)/**/*-test.js
```

Comment:
We can switch to no -test.js but then it should go in another PR (moving all -test.js), for consistency